src/common/zscore/cgmzscore/src/table.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+
+class Table:
+
+    def __init__(self, table_file: str):
```

Comment:
I see you passing pathlib.Path to this when using this

```suggestion
    def __init__(self, table_file: Path):
```

Reply:
I was getting flake8 error while using Path.

Reply:
Import pathlib